extensions/BugModal/web/bug_modal.js

```diff
@@ -1335,7 +1333,7 @@ function confirmUnsafeURL(url) {
 // fix url after bug creation/update
 if (history && history.replaceState) {
     var href = document.location.href;
-    if (!href.match(/show_bug\.cgi/)) {
+    if (!href.match(/new-bug/) && !href.match(/show_bug\.cgi/)) {
```

Comment:
instead of checking the url, let's add an element with a data attribute and check that.
Not sure what element this should be, but let's chat about it tomorrow.